src/Main.cpp

```diff
@@ -77,6 +77,7 @@ int main(int argc, char** argv) {
 
     opts.add_options("General")
         ("h,help", "Print help message and exit")
+        ("j,json", "Print output in JSON Format")
```

Comment:
Since we may want to add more formats in future, e.g. CSV, let's replace it with `-f` / `--format` with allowed values `text` and `json`. The short usage would be: `-fjson`, which is still compact.